### PR TITLE
Google login

### PR DESCRIPTION
**Pull request by sunnybrie**
Let me know what you think of the login. 

[Ticket Link](https://trello.com/c/BXCoWEbB)

### CODE REVIEW

File: app/src/Account.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from "react";
```
Comment (aedward8) [inline app/src/Account.js:1]: useState is not used currently so its throwing me some warnings in the console~~ but im sure you will use it later but just a reminder :)!

Comment (sunnybrie) [reply app/src/Account.js:1]: Thanks for the heads up!

File: app/src/App.js [hunk 2]
```diff
@@ -1,56 +1,38 @@
...
-      <TaskList tasks={tasks} />
-      <AddTask loadTasks={loadTasks} />
+      <Account
+        user={user}
+        setUser={(e) => setUser(e)}
+        logOut={() => logOut()}
+        isSignedIn={isSignedIn}
+        setIsSignedIn={(e) => setIsSignedIn(e)}
+      />
+      {isSignedIn ? <Planner /> : <p>Yo, sign in already.</p>}
```
Comment (aedward8) [inline app/src/App.js:33]: loving the use of the ternary here!

**Review by aedward8 — APPROVED** [review]
I logged in and logged out successfully! woohoo! so proud you got this working!! I liked your code it is already pretty concise looking to be honest.

I got this warning "[Violation] 'setInterval' handler took 73ms" but it still compiled & worked but maybe for a future issue this can be fixed.

Amazing job sunnyBrie!

**Pull request closed without merging**